internal/future/statestore/redis.go

```diff
@@ -89,3 +91,33 @@ func redisCloser(closer func() error) {
 		}).Info("failed to close redis")
 	}
 }
+
+// CreateTicket creates a new Ticket in the state storage. This method fails if the Ticket already exists.
+func (rb *redisBackend) CreateTicket(context.Context, pb.Ticket, int) error {
+	return fmt.Errorf("not implemented")
+}
+
+// GetTicket gets the Ticket with the specified id from state storage. This method fails if the Ticket does not exist.
+func (rb *redisBackend) GetTicket(context.Context, string) (*pb.Ticket, error) {
+	return nil, fmt.Errorf("not implemented")
+}
+
+// DeleteTicket removes the Ticket with the specified id from state storage. This method succeeds if the Ticket does not exist.
+func (rb *redisBackend) DeleteTicket(context.Context, string) error {
+	return fmt.Errorf("not implemented")
```

Comment:
Errorf implies that there's parameters. Refer using grpc errors here anyway.

Reply:
Done!